src/app-layout/index.tsx

```diff
@@ -364,7 +463,8 @@ const OldAppLayout = React.forwardRef(
       toolsPadding:
         // tools padding is displayed in one of the three cases
         // 1. Nothing on the that screen edge (no tools panel and no split panel)
-        (toolsHide && (!splitPanelDisplayed || finalSplitPanePosition !== 'side')) ||
+        toolsHide ||
+        (drawers && !activeDrawerId && (!splitPanelDisplayed || finalSplitPanePosition !== 'side')) ||
```

Comment:
It looks like now there is a duality between the old `tools` and the new `drawers` API. 

I think, we should resolve this duality in one function, like `applyToolsStateToDrawers(drawers, {toolsHide, toolsOpen, tools})` and then use a single `rightDrawer` (or any better name) abstraction instead of repeat `tools || drawers` many times

Reply:
I'm a bit confused by this - I agree with the sentiment but let's discuss implementation more.

Reply:
Notes from live discussion: to-do in a follow-up.